core/src/main/java/hudson/slaves/SlaveComputer.java

```diff
@@ -150,7 +150,7 @@ private OutputStream decorate(OutputStream os) {
             try {
                 os = f.decorateLogger(this,os);
             } catch (IOException|InterruptedException e) {
-                LOGGER.log(Level.WARNING, "Failed to filter log with "+f, e);
+                logger.log(Level.WARNING, "Failed to filter log with "+f, e);
```

Comment:
I thing LOGGER was legitimate in this case, because it is not related to slave logs